#Array of GroupProperties data structure
GroupProperties[] array